### DISCUSSION TITLE
MQTT encode quality - higher or lower better?

### DISCUSSION THREAD

**Discussion by mattgmoser** [blakeblackshear/frigate#2227] [Ask A Question]
Hi all. I have a question regarding MQTT configuration and the quality setting. What value is the best encode quality I can set this to?

Looking at the documentation, 'live' and 'birdseye' both have a quality field from 1-31, where 1 is the highest. MQTT seems to be on a different scale. Thanks!

**Comment by blakeblackshear** [comment]
That scale refers to the jpg encode quality. You can set it up to 100. Anything past 90 is probably diminishing returns, but you should experiment.

**Reply by zoltanszalontay** [reply]
Thanks a lot for this clarification! Now I understand why my push messages contain mediocre images. Fixed now.
